R/bm-read-csv.R

```diff
@@ -13,10 +13,10 @@ read_csv <- Benchmark(
   setup = function(source = names(known_sources),
                    reader = "arrow",
                    compression = c("uncompressed", "gzip"),
-                   output = c("arrow_table", "data_frame")) {
+                   output_format = c("arrow_table", "data_frame")) {
     reader <- match.arg(reader, c("arrow", "data.table", "vroom", "readr"))
     compression <- match.arg(compression)
-    output <- match.arg(output)
+    output_format <- match.arg(output_format)
```

Comment:
I'm totally fine with this change, though we should confirm that it won't bork ursacomputing/benchmarks calling of these.

https://github.com/ursacomputing/benchmarks/blob/f1114e0442de39239ca80d41b7e377598e4692dc/benchmarks/csv_benchmark.py#L44

Reply:
Good call. It may keep working regardless because of partial matching? But we should change it anyway.

Reply:
PR: https://github.com/ursacomputing/benchmarks/pull/104